telemetry/csharp/AwsToolkit.Telemetry.Events.Generator.Tests/test-data/expectedCode-supplemental.txt

```diff
@@ -322,11 +322,21 @@ namespace Test
     /// Emits milliseconds
     public sealed class EmitSomeIntMs : BaseTelemetryEvent
     {
+        
+        public EmitSomeIntMs()
+        {
+            this.Passive = false;
+        }
     }
     
     /// Emits something without a unit defined
     public sealed class EmitNoUnit : BaseTelemetryEvent
     {
+        
```

Comment:
nit: why is there an empty line at the start but not at the bottom? looks funky

Reply:
Its an artifact of the code generation library.